proposals/0024-repair-and-restart.md

```diff
@@ -0,0 +1,185 @@
+---
+simd: '0024'
+title: Repair and start in a cluster restart
+authors:
+  - Wen Xu (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2023-04-07
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+Improve the current [cluster restart](https://docs.solana.com/running-validator/restart-cluster)
+procedure such that validators can automatically figure out confirmed slots
```

Comment:
```suggestion
procedure such that validators can automatically figure out the highest optimistically confirmed slot
```

Reply:
I think this is accurate right? 

For now, I think we've whittled this SIMD down to the programatic determination of the  highest optimistically confirmed slot, after which essentially the existing restart mechanism is used: ie, shred-version changes, hard fork at that oc slot, all pre-restart blocks above the oc slot are flushed. Am I on the same page still?

Reply:
Yes, it should be highest oc slot, thanks for pointing that out.